### DISCUSSION TITLE
What is the latest version of ARPACK?

### DISCUSSION THREAD

**Discussion by Siraj19176** [opencollab/arpack-ng#466] [Q&A]
Hi,

The file "version.h" at the following link:
https://github.com/opencollab/arpack-ng/blob/master/SRC/version.h
mentions that the "VERSION_NUMBER" is 2.1. 
However, in the release tags, I can see that there are newer upstream releases, such as 3.9.0
https://github.com/opencollab/arpack-ng/releases/tag/3.9.0.

My question is: Is "version.h" out of date, and what is the latest version of ARPACK?

**Comment by fghoussen** [comment]
`arpack` (the algorithm) is an old scary thing that is no more maintained by the guys who developed it, but, that is essential for many things (math, ML, ...): I guess 2.1. is the latest version of that part of the repo (i.e. version of the code initially forked).

`arpack-ng` (the dev-ops bundle around the algorithm - cmake, ...) is maintained by full-open-source volunteers to keep `arpack` in a shape that can be used on modern arch / compilers: I guess 3.9.1. is the latest version of that part of the repo.

@sylvestre: what do you say?
